portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,43 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+var slideIndex = 1;
```

Comment:
- would it be better to have the gallery javascript in a separate file (e.g. `gallery.js`)?
- almost all code I see (except Matlab) starts indexing with 0 and not with 1
- global variables is something to avoid (even though you will have at least one in this case, but once you add the second gallery-specific global variable, I'd create an object instead to encapsulate all state)